Translation Script/Map107.csv

```diff
@@ -110,50 +110,50 @@ one more day. If only...)"
 "ビンが割れている。
 アルコールの匂いがする・・・","This bottle is smashed.
 It smells like alcohol..."
-"\n<\n[1]>（勿体ない・・・）","\n<\n[1]>(Such a waste...)"
-"EV017","EV017"
-"リリーの日記",""
+"\n<\n[1]>（勿体ない・・・）",\n<\n[1]>(Such a waste...)
+"EV017",EV017
+"リリーの日記",Lily's Diary
 "リリーの日記
 ミスト屋敷襲撃","Lily's Diary
 Misty Mansion Attack"
-"読んでみる","Read"
-"やめておく","Maybe not."
-"EV019","EV019"
-"ニンニクとお肉。","Meat and garlic."
-"\n<\n[1]>（料理をしている時間はないね。）","\n<\n[1]>（This isn't the time to be cooking.）"
+"読んでみる",Read
```

Comment:
Oh dear, we can't have this. If we remove the commas like this, the csv will break!

That wouldn't be good at all. Definitely go through and fix these if you can haha